checksum/src/main/resources/application.conf

```diff
@@ -0,0 +1,16 @@
+sqs {
+    endpoint = "https://sqs.eu-west-2.amazonaws.com"
+    queue {
+        input = "set_me_input_queue"
```

Comment:
Is there a default input value to use running locally?

Reply:
No, we don't have any way of doing local dev with local sqs queues at the moment and I don't think we're planning on it yet. The reason I haven't put the real values from intg in is because it's got the account number in the queue url which I don't want to put on Github.

I've added a README with some instructions on running it locally. They're a bit hacky, let me know what you think.